integration-v2.groovy

```diff
@@ -0,0 +1,825 @@
+                                stage("Deploy ${stageId}") {
+                                    try {
+                                        withCredentials([
+                                            [
+                                                $class: 'AmazonWebServicesCredentialsBinding',
+                                                credentialsId: awsCred,
+                                                accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                                                secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+                                            ]
+                                        ]) {
+                                            String pwd = sh(script: "pwd", returnStdout: true).trim()
+                                            // Login to Docker registry
+                                            sh """
+                                                echo ${dockerRegistryPasswordSafe} | sudo docker login ${dockerRegistrySafe} --username ${dockerRegistryUsernameSafe} --password-stdin
+                                            """
+
+                                            dir("${patternDirSafe}") {
+                                                def dbWriterEndpointsJson = sh(script: "terraform output -json | jq -r '.database_writer_endpoints.value'", returnStdout: true).trim()
+                                                def dbWriterEndpoints = new groovy.json.JsonSlurperClassic().parseText(dbWriterEndpointsJson)
+                                                if (!dbWriterEndpoints) {
+                                                    error "DB Writer Endpoints are null or empty for ${patternDirSafe}. Please check the Terraform output."
+                                                }
+                                                println "DB Writer Endpoints: ${dbWriterEndpoints}"
+                                                // Convert LazyMap to HashMap
+                                                patternSafe.dbEndpoints = new HashMap<>(dbWriterEndpoints)
+
+                                                def (endpoint, dbPort) = patternSafe.dbEndpoints["${dbEngineNameSafe}-${dbEngineList[dbEngineNameSafe].version}"]?.tokenize(':')
+                                                def namespace = "${patternSafe.id}-${dbEngineNameSafe}"
+                                                sh """
+                                                    # Change context
+                                                    kubectl config use-context ${patternDirSafe}
+
+                                                    # Create a namespace for the deployment
+                                                    kubectl create namespace ${namespace} || echo "Namespace ${namespace} already exists."
+
+                                                    aws s3 cp --quiet s3://${tfS3Bucket}/tools/client-truststore.jks .
```

Comment:
Is it possible to remove this step? We regenerate the keystores for every release. If we follow this pattern, we will have to store those in S3 for every release.